shell/platform/android/io/flutter/embedding/android/FlutterSurfaceView.java

```diff
@@ -135,6 +136,28 @@ private void init() {
     setAlpha(0.0f);
   }
 
+  // This is a work around for TalkBack.
+  // If Android decides that our layer is transparent because, e.g. the status-
+  // bar is transparent, TalkBack highlighting stops working.
+  // Explicitly telling Android this part of the region is not actually
+  // transparent makes TalkBack work again.
+  // See https://github.com/flutter/flutter/issues/73413 for context.
+  @Override
+  public boolean gatherTransparentRegion(Region region) {
+    if (getAlpha() < 1.0f) {
+      return false;
+    }
+    final int[] location = new int[2];
+    getLocationInWindow(location);
+    region.op(
```

Comment:
did you intend to read the output of this?

Reply:
The CL this was based on didn't use the return value. I'm not sure it's important here - I guess we could just return this value instead of returning true, but this op should always modify the region as long as the surface view is actually being used.